llvm/tools/llvm-gsymutil/Opts.td

```diff
@@ -16,6 +16,8 @@ def verbose : FF<"verbose", "Enable verbose logging and encoding details">;
 defm convert :
   Eq<"convert",
      "Convert the specified file to the GSYM format.\nSupported files include ELF and mach-o files that will have their debug info (DWARF) and symbol table converted">;
+def store_merged_function_info :
+  FF<"store-merged-function-info", "Store all functions with overlapping address ranges in the GSYM, instead of keeping only one per range (default).">;
```

Comment:
I would name this `"merged-functions"`, the current one is a bit long. We also want to make sure we are clear here with the description: `"Encode merged function information for functions in debug info that have matching address ranges. Without this option one function per unique address range will be emitted."`